README.md

```diff
@@ -31,6 +31,24 @@ You can check your node and npm versions by typing the mentioned commands in the
 node -v
 npm -v
 ```
+### For Windows Users only
+TO run the project locally on your device you need to make the following changes
+   1. Got to /package.json
+   2. Find the "scripts" object in the 5th line
+   3. In the 6th and 7th line replace the following code 
+      {
+         "serve": "vue-cli-service serve",
+         "build": "vue-cli-service build",
+      }
+
+      with
+
+      {
+         "serve": "set NODE_OPTIONS=--openssl-legacy-provider && vue-cli-service serve",
+         "build": "set NODE_OPTIONS=--openssl-legacy-provider && vue-cli-service build",
+      }
```

Comment:
@Kushdapush Why do we need the change? What's the error we are getting on the windows? 

cc @arminpatel @Tusharjain123

Reply:
sir when a user tries to run the code on windows , then they encounter this error
![WhatsApp Image 2023-07-09 at 00 27 53](https://github.com/bsoc-bitbyte/BSoC-Website/assets/129064700/eac99324-3883-4f1f-84d2-f08bf6b75ad0)

Reply:
and after doing the above mentioned change , the code runs locally without any issue,
this problem was encountered by many of my batchmates while working on this project

Reply:
Why dont you change this in the package.json directly and raise a PR instead of adding it in the readme.md ?

Reply:
Sir I think if we change the code directly from package.json , it may lead to unexpected behaviour for other OS users (linux mainly), Also it was told by some maintainer at the start of the programe that we dont have to push package.json files


But if you want, then ill add the changes to package.json directly

Reply:
@Kushdapush can you tell me which node version you are using?

Reply:
@Tusharjain123 19.8.1 sir

Reply:
try with node version 14 or 16. You will not get this error.
You can use nvm for switching the node version.

Reply:
Ok sir , so shall i update the readme with this information that you provided, since the main aim of this issue was to resolve the error that a contributor gets while running the code and this problem is occuring with nearly all the contributors working on windows so we cant tell them all to do this indivisually

Reply:
Yeah, you can
What you say @Priyansh61

Reply:
@Tusharjain123  @Priyansh61  @JawHawk 
Sir shifting to an older node version may not be an appropriate solution to this problem as it would take a toll on the overall working proficiency of the development server, also why would somebody switch to a less effective node version when they can get the same work done on the latest version just by making some changes in the way the code is to be run

Reply:
Ok sir, in my latest commit, I have made the changes in package.json file, 
Please review them

Reply:
@Priyansh61 @Tusharjain123 sir any update on this ?

Reply:
@Kushdapush Please remove the changes from readme.md

Reply:
@Priyansh61 sir i already did that in my 2nd commit please check again

Reply:
![image](https://github.com/bsoc-bitbyte/BSoC-Website/assets/129064700/6dec5d44-c3f5-414e-89ed-f5de2e94f7b3)

Reply:
here's the screenshot of the 2nd commit